app/src/main/java/com/example/synerzip/recircle_android/models/user_messages/OwnerProdRelatedMsg.java

```diff
@@ -15,7 +18,7 @@
  */
 @Getter
 @Setter
-public class OwnerProdRelatedMsg {
+public class OwnerProdRelatedMsg implements Parcelable {
```

Comment:
We can make a single model class for Owner & Renter Messages as the structure is same?